packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -296,7 +298,7 @@ const SelectableSearchInput = (props: TSelectableSearchInputProps) => {
     componentName: 'SelectableSearchInput',
   });
 
-  const { onFocus, onBlur, name } = props;
+  const { onFocus, onBlur, name, onChange } = props;
```

Comment:
If we want to destructure the `onChange` prop, I think we should use the destructured value everywhere in the file. Currently we have been using `props.onChange` in other parts of the file and we should be consistent.